infra/main.bicep

```diff
@@ -0,0 +1,483 @@
+targetScope = 'resourceGroup'
+
+@minLength(1)
+@maxLength(64)
+@description('Name of the the environment which is used to generate a short unique hash used in all resources.')
+param environmentName string
+
+param resourceToken string = toLower(uniqueString(subscription().id, environmentName, location))
+
+@description('Location for all resources.')
+param location string
+
+@description('Name of App Service plan')
+param hostingPlanName string = '${environmentName}-hosting-plan-${resourceToken}'
+
+@description('The pricing tier for the App Service plan')
+@allowed([
+  'F1'
+  'D1'
+  'B1'
+  'B2'
+  'B3'
+  'S1'
+  'S2'
+  'S3'
+  'P1'
+  'P2'
+  'P3'
+  'P4'
+])
+param hostingPlanSku string = 'B3'
+
+@description('Name of Web App')
+param websiteName string = '${environmentName}-website-${resourceToken}'
+
+@description('Name of Application Insights')
+param applicationInsightsName string = '${environmentName}-appinsights-${resourceToken}'
+
+@description('Use semantic search')
+param azureSearchUseSemanticSearch string = 'false'
+
+@description('Semantic search config')
+param azureSearchSemanticSearchConfig string = 'default'
+
+@description('Is the index prechunked')
+param azureSearchIndexIsPrechunked string = 'false'
+
+@description('Top K results')
+param azureSearchTopK string = '5'
+
+@description('Enable in domain')
+param azureSearchEnableInDomain string = 'false'
+
+@description('Content columns')
+param azureSearchContentColumns string = 'content'
+
+@description('Filename column')
+param azureSearchFilenameColumn string = 'filename'
+
+@description('Title column')
+param azureSearchTitleColumn string = 'title'
+
+@description('Url column')
+param azureSearchUrlColumn string = 'url'
+
+@description('Name of Azure OpenAI Resource')
+param azureOpenAIResourceName string = '${environmentName}-openai-${resourceToken}'
```

Comment:
If the original project uses OpenAI, then use that across the board.  If not, then use OpenAi, just be consistent

Reply:
Okay, the original project uses OpenAI.